### PR TITLE
Merge CORS headers + Allow modifying app settings from pytest marks

### PR DESCRIPTION
**Pull request by Qiwn**
https://github.com/plone/guillotina/pull/585

### CODE REVIEW

**Comment by vangheem** [comment]
Hi @Qiwn! Looks like we have a test failure. Can you check?

https://travis-ci.org/plone/guillotina/jobs/558827073#L1317

File: docs/source/developer/testing.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Testing
+
+## Customizing app settings on tests
+
+You can use pytest markers to overwrite specific application settings
+on tests, for instance:
+
+```python
+@pytest.mark.app_settings({
```
Comment (vangheem) [inline docs/source/developer/testing.md]: I think maybe a bad rebase here? We don't ahve this in 4.x!

Comment (vangheem) [reply docs/source/developer/testing.md]: Oh, I see. It's backported now! Awesome!

Comment (Qiwn) [reply docs/source/developer/testing.md]: :)

**Pull request merged**